scripts/create-cert.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Since this is for digicert, suggestion: rename the file to digicert-create-cert ?